### PR TITLE
Fixes small typo on users controller(admin)

### PR DESCRIPTION
**Pull request by ingedmundo**
Fixes a small typo on admin/users_controller.
Fixes a wrong url redirection, now it redirects to the users dashboard.

### CODE REVIEW

**Comment by vicmaster** [comment]
Check broken test

**Comment by mumoc** [comment]
As Victor said, check broken test

**Comment by vicmaster** [comment]
LGTM

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -23,12 +23,12 @@ def sort_direction
   end
 
   def restrict_access
-    redirect_to session[:last_moudle] unless current_user.
+    redirect_to current_user unless current_user.
```
Comment (mumoc) [inline app/controllers/admin/users_controller.rb]: :/ this sounds .... weird and was kinda "hard to read", I would create a helper method called "autorized?" or something alike and refactor this as:
```
...
redirect_to(current_user) if authorized?
...

def authorized?
   current_user.is_admin || current_user.is_badges_admin?
end
```

even further, you could move the whole logic to the user ... I mean, reviewing the ```roles.include?``` at controller level sounds like knowing too much; the other way would be:
```
def restrict_access
  redirect_to current_user if current_user.is_bades_admin?
end
```

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -23,12 +23,12 @@ def sort_direction
...
       roles.include?(badges_admin) || current_user.is_admin
   end
 
   def get_last_module
```
Comment (mumoc) [inline app/controllers/admin/users_controller.rb]: you could call this method just as ```last_module``` ....

File: app/decorators/controllers/badges_engine/application_controller_decorator.rb [hunk 3]
```diff
@@ -4,12 +4,12 @@
   private
 
   def restrict_access
-    redirect_to session[:last_moudle] unless
+    redirect_to session[:last_module] unless
```
Comment (mumoc) [inline app/decorators/controllers/badges_engine/application_controller_decorator.rb]: in the other controller you changed this to current_user, why the difference here, also, seeing this, I would recommend you to create a concern.... module.... service maybe.... or just a helper method for this two methods as you're currently duplicating logic

File: vendor/engines/workshops_engine/app/controllers/workshops_engine/application_controller.rb [hunk 4]
```diff
@@ -7,12 +7,12 @@ class ApplicationController < ::ApplicationController
     private
 
     def restrict_access
-      redirect_to session[:last_moudle] unless
+      redirect_to main_app.user_path(current_user.id) unless
```
Comment (mumoc) [inline vendor/engines/workshops_engine/app/controllers/workshops_engine/application_controller.rb]: all these controllers makes me think you would need to implement a better way to restrict access... I mean, maybe just creating some kind of module then extending the controllers with it, or making use of inheritance ... don't know, but surely something could be done (given the level of duplication you have)

**Comment by vicmaster** [comment]
This PR is going to be merged as it is right now... I created this task https://trello.com/c/pv2x5ZFV/117-refactor-user-controller assigned to @ingedmundo to make that refactor. Thanks @mumoc

**Comment by mumoc** [comment]
:shipit:

**Pull request merged**